src/backend/updates.rs

```diff
@@ -672,6 +711,16 @@ fn check_candid_interface_compatibility() {
     );
 }
 
+#[update]
+fn backup() {
```

Comment:
A malicious stalwart can drain cycles and DoS the canister by calling `archive` and `backup` in a loop. Is this a risk we are willing to accept?